meta-lxatac-software/recipes-core/bundles/files/hook.sh

```diff
@@ -44,7 +44,22 @@ function migrate () {
 	cp -a "$1" "${RAUC_SLOT_MOUNT_POINT}/$1"
 }
 
+function mock_resize2fs () {
+	# "old" bundles (~2023-10) do not have a resize2fs,
+	# making the resize phase of the bundle install fail.
+	# Provide a "resize2fs" so the installation works at least.
+	# Can be removed once all TACs are updated to have resize2fs.
+	if [ ! -f "/sbin/resize2fs" ]; then
+		echo "#!/bin/sh" > /sbin/resize2fs
+		echo "echo \"Dummy resize2fs installed by RAUC hook\"" >> /sbin/resize2fs
+		chmod +x /sbin/resize2fs
+	fi
+}
+
 case "$1" in
+	slot-pre-install)
+		mock_resize2fs
+		;;
```

Comment:
Drop this change and update the commit message.

Reply:
Ooops, looks like I've opened the pull request from a branch here and not from by fork.
I've updated the branch here now.